src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -688,7 +688,7 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('enabled_locales')
-                            ->prototype('scalar')
+                            ->prototype('scalar')->end()
                             ->defaultValue([])
```

Comment:
`->defaultValue([])` could even be removed entirely here. It is already the default value for a prototyped array node. We don't need to move the broken default value to the `enabled_locales` node.

Reply:
We have a lot of occurences of `->defaultValue([])`on prototyped array nodes. Couldn't we clean them all starting from a lower branch instead?

Reply:
Let's do that in another PR indeed.

Reply:
Well, but here, you are actually adding it on that node (as the code was previously on the scalar node).